docs/en/api/getUsersByGroup.md

```diff
@@ -36,6 +36,7 @@ __Throttle Limits__: Maximum 25 requests per minute per a client. See [Throttlin
 | X-Request-Id | header | false | {% include_relative partials/requestIdDescription.md %} |
 | directOnly | query | false | {% include_relative partials/directOnlyDescription.md %} |
 | status | query | false | For product profiles only, return only active or inactive members. Pass `active` to list users that have been provisioned for the product and have an active license. Pass `inactive` to list users who have been added to the product profile but do not have an _active_ license. When not provided, lists all member users regardless of their entitlement status.|
+| excludeGroups | query | false | Default value is false. When true is passed the response will exclude the `groups` array from being returned for each individual user. See [example](#getUsersWithNoGroupsExample). |
```

Comment:
Worth escaping the `false` and `true` values to make them stand out (as you have with `groups`)